sources/packages/web/src/assets/css/global-style-variables.scss

```diff
@@ -1,6 +1,6 @@
 /** Brand Colors - Figma **/
 $brand-blue: #2965c5;
-$brand-grey: #69768C;
+$brand-grey: #69768c;
```

Comment:
what is the change here

Reply:
This is the prettier formatting of CSS. It is always using uppercase for alphabets when it formats.